flepimop/main_scripts/inference_slot.R

```diff
@@ -625,20 +631,20 @@ for(seir_modifiers_scenario in seir_modifiers_scenarios) {
       # run
       if (config$inference$do_inference){
         sim_hosp <- flepicommon::read_file_of_type(gsub(".*[.]","",this_global_files[['hosp_filename']]))(this_global_files[['hosp_filename']]) %>%
-          dplyr::filter(time >= min(obs$date),time <= max(obs$date))
-          
-          # add aggregate groundtruth to the obs data for the likelihood calc
-          if (opt$incl_aggr_likelihood){
+          dplyr::filter(time >= min(obs$date), time <= max(obs$date))
```

Comment:
hm actually should this be filtering date? i see it is the same in both branches though @jcblemai